litgpt/prompts.py

```diff
@@ -272,6 +272,17 @@ def apply(self, prompt: str, **kwargs: str) -> str:
         return f"Instruct: {prompt}\nOutput:"
 
 
+class Phi3(PromptStyle):
+    def apply(self, prompt: str, **kwargs: str) -> str:
+        return (
+            "<|system|>\n"
```

Comment:
They just changed this: https://huggingface.co/microsoft/Phi-3-mini-4k-instruct/commit/688dd7b82eb7ff6cb786a9f723ab3e35479a19d5